benchmarks/benchmark.rs

```diff
@@ -0,0 +1,260 @@
+//! A script for running benchmarks on a canister.
```

Comment:
nit: Should we add an example usage of the script (e.g. with the arguments that we need in the average case).

Reply:
Done. It's just `cargo bench` by the way.